### PR TITLE
doc: add how to run application in QEMU on Windows

### PR DESCRIPTION
**Pull request by michal-smola**
Documentation does not describe how to run QEMU on Windows. Add description of steps needed in Beyond the Getting Started Guide and in Application Development sections.
Tested using Windows 11 and QEMU installer from https://qemu.weilnetz.de/w64/

### CODE REVIEW

**Comment by michal-smola** [comment]
Any comments, please?

File: doc/develop/application/index.rst [hunk 1]
```diff
@@ -1011,10 +1011,15 @@ for additional information on how to flash your board.
...
-is not yet supported on Windows). It allows you to run and test an application
-virtually, before (or in lieu of) loading and running it on actual target
-hardware. Follow these instructions to run an application via QEMU:
+The kernel has built-in emulator support for QEMU (on Linux/macOS only).
+It allows you to run and test an application virtually, before
+(or in lieu of) loading and running it on actual target hardware.
+
+On Windows, you need to install QEMU manually from
+`Download QEMU <https://www.qemu.org/download/#windows>`_. After installation,
+add path to QEMU installation folder to PATH environment variable.
```
Comment (henrikbrixandersen) [inline doc/develop/application/index.rst:1020]: Not sure "the kernel" is used correctly here. I'd rephrase it to say that "Zephyr has ..." and remove the "(on Linux/macOS only)".

Also, I think mentioning how to install QEMU in the beyond-GSG is sufficient. No need to repeat it here.

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

**Review by henrikbrixandersen — DISMISSED** [review]

File: doc/develop/application/index.rst [hunk 2]
```diff
@@ -1011,10 +1011,13 @@ for additional information on how to flash your board.
 Running in an Emulator
 ======================
 
-The kernel has built-in emulator support for QEMU (on Linux/macOS only, this
-is not yet supported on Windows). It allows you to run and test an application
-virtually, before (or in lieu of) loading and running it on actual target
-hardware. Follow these instructions to run an application via QEMU:
+Zephyr has built-in emulator support for QEMU.
+It allows you to run and test an application virtually, before
+(or in lieu of) loading and running it on actual target hardware.
```
Comment (kartben) [inline doc/develop/application/index.rst:1016]: formatting is off here, but in fairness we have less than ideal infrastructure to help enforce our "100 columns" rule in docs (see #68430)

**Review by kartben — DISMISSED** [review]

**Review by PerMac — CHANGES_REQUESTED** [review]
I am requesting changes to align the documentation with required changes from #67595:
- add a statement that QEMU_BIN_PATH has to be added to the system's variables to enable qemu in twister

**Comment by michal-smola** [comment]
Added statement about QEMU_BIN_PATH for Twister in Beyond the Getting Started Guide.

**Review by PerMac — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]

**Comment by fabiobaltieri** [comment]
@carlescufi ping

**Review by kartben — APPROVED** [review]
It happens that I needed this recently and it worked perfectly, thanks!
